samples/js/fb/find-photos/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "noms-fb-find-photos",
-  "description": "Finds photos in slurped flickr data",
+  "name": "noms-facebook-find-photos",
```

Comment:
ok, changed it back, but then changed noms-facebook-slurp to be noms-fb-slurp to match.